src/utils/helpers/formatDate.js

```diff
@@ -0,0 +1,2 @@
+export const formatDate = (date) =>
+  `${date.getDate()}.${date.getMonth() + 1}.${date.getFullYear()}`;
```

Comment:
In the figma design the date is in `dd.mm.yy` format, but you are showing it in `dd.mm.yyyy` format